mmselfsup/datasets/builder.py

```diff
@@ -53,7 +53,7 @@ def build_dataloader(dataset,
                      replace=False,
                      seed=None,
                      pin_memory=True,
-                     persistent_workers=True,
+                     persistent_workers=False,
```

Comment:
why do you change this?

Reply:
Thank you for your valuable reviews. For the convenience of debugging, I change the default value of `persistent_worker` and forgot to change it back in my official PR. I will fix it~